build.gradle

```diff
@@ -54,17 +55,17 @@ compileJava {
 }
 
 dependencies {
-    compile('org.seleniumhq.selenium:selenium-java:3.4.0'){
+    compile('org.seleniumhq.selenium:selenium-java:3.5.1'){
```

Comment:
Yeah just tried this like below but it still downloads latest sub modules of it.
```
compile('org.seleniumhq.selenium:selenium-java:3.4.+'){
        exclude module: 'cglib'
        exclude group: 'com.google.code.gson'
        force = true
    }
```
It still downloads latest `selenium-api` i.e 3.5.2